alphastats/gui/pages/02_Import Data.py

```diff
@@ -34,9 +38,14 @@
 if "loader" not in st.session_state:
     st.session_state["loader"] = None
 
+if "gene_to_prot_id" not in st.session_state:
+    st.session_state["gene_to_prot_id"] = {}
 
-def load_options():
+if "organism" not in st.session_state:
+    st.session_state["organism"] = 9606
```

Comment:
maybe add a comment what this magic organism is? :-)

Reply:
Done